### ISSUE TITLE
batches view: slider: do not slide automatically

### ISSUE THREAD

**Issue by jgehrcke** [conbench/conbench#551]
A person looking at a plot like shown below has ~5 seconds of time before the next plot slides into the view :). Disable that auto-sliding!

![image](https://user-images.githubusercontent.com/265630/208668656-37fafc6d-bc67-4d8a-8176-963ddd4040ad.png)

**Comment by jgehrcke** [comment]
Also: the sliding effect seems to happen when there's just one item in the slider (i.e., it slides from the same plot to the same plot).

Action item: disable slider entirely when there is just one plot to show.

**Comment by jonkeane** [comment]
Agreed that the sliding is a bit too aggressive. I would very much be in favor of disabling this for now
